### PR TITLE
Mav 08 game function

### PR DESCRIPTION
**Pull request by Mhaventhan**
ball movement physics.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -5,43 +5,67 @@ body{
...
+  color: white;
+  border-radius: 6%;
+  width: 200px;
+
+
 }
 #game {
     position: relative;
     height: 500px;
     top: 20px;
```
Comment (richgurney) [inline index.css]: Check out your indentation here

File: index.js [hunk 2]
```diff
@@ -25,68 +25,129 @@ $(function() {
...
-  paddle = $('.paddle'),
-  paddle_1 = $('#paddle_1'),
-  paddle_2 = $('#paddle_2'),
-  restart = $('#restart_btn');
+  var game = $('#game');
+  const $ball = $('#ball');
+  var paddle = $('.paddle');
+  var paddle_1 = $('#paddle_1');
+  var paddle_2 = $('#paddle_2');
+  const restart = $('#restart_btn');
```
Comment (richgurney) [inline index.js]: What you using the const for here?

File: index.js [hunk 3]
```diff
@@ -95,65 +156,52 @@ $(function() {
...
-    if (b1 < y2 || y1 > b2 || r1 < x2 || x1 > r2) return false;
-    return true;
-}
-
-
-
+      });
+      // Collision detection algorithm
+      function collision($div1, $div2) {
+        var x1 = $div1.offset().top;
```
Comment (richgurney) [inline index.js]: These need to be more readable

File: index.css [hunk 4]
```diff
@@ -29,8 +30,6 @@ h1{
   color: white;
   border-radius: 6%;
   width: 200px;
-
-
 }
 #game {
```
Comment (AEJackson) [inline index.css]: you haven't line up your indentations
all tag groups should be indented the same

**Pull request merged**